cmd/manager/main.go

```diff
@@ -192,7 +202,8 @@ func parseConfiguration(log *zap.SugaredLogger) Config {
 	flag.BoolVar(&config.EnableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-
+	flag.StringVar(&config.LogLevel, "log-level", "info", "Log level. Available values: debug | info | warn | error | dpanic | panic | fatal")
```

Comment:
log-level and log-encoder should be added to the pod configuration manager.yaml as well.

Reply:
Fixed

Reply:
Should files under deploy and bundle sub-directories be updated also?